system-parachains/asset-hubs/asset-hub-polkadot/src/weights/pallet_election_provider_multi_block_unsigned.rs

```diff
@@ -42,6 +42,7 @@
 #![allow(unused_parens)]
 #![allow(unused_imports)]
 #![allow(missing_docs)]
+#![allow(dead_code)]
```

Comment:
we do not have this in template? right now we always have to add this manually?

Reply:
It is actually in the template, maybe as you generated the weights the last time it was not yet part of it.